GitUp/Application/Document.m

```diff
@@ -1838,12 +1826,13 @@ - (IBAction)editConfiguration:(id)sender {
 
 - (void)_setSearchFieldPlaceholder:(NSString*)placeholder {
   // 10.11 and earlier: search placeholders have the same length to work around incorrect centering.
-  if (kCFCoreFoundationVersionNumber < kCFCoreFoundationVersionNumber10_12) {
+  if (@available(macOS 10.12, *)) {
+  } else {
```

Comment:
Maybe if `!@available`?

Reply:
Sadly, I’m like 99% sure it won’t work having tried it before. But once I get back to my desk, I’ll double-check. :)

Reply:
Yeah, it does not. `@available does not guard availability here; use if (@available) instead`